backend/internal/common/env_config.go

```diff
@@ -145,10 +160,21 @@ func parseEnvConfig() error {
 	case "":
 		EnvConfig.KeysStorage = "file"
 	case "database":
-		// If KeysStorage is "database", a key must be specified
-		if EnvConfig.EncryptionKey == "" && EnvConfig.EncryptionKeyFile == "" {
+		// Resolve encryption key using the same pattern
+		encryptionKey, err := resolveStringOrFile(
```

Comment:
I had not done this for the encryption key because it's managed separately: https://github.com/pocket-id/pocket-id/blob/d12c1a0dc386f70c582821968ec0ed554bad0aa1/backend/internal/utils/jwk/utils.go#L49

I would prefer to keep this separate because it's very delicate. For example, the code added above for trimming whitespace (see my other comment) is very dangerous here, as would cause key files to be read differently, which would make keys fail to decrypt.